### PR TITLE
[ACM-4126]: Adding docs about creating a hosted cluster in multiple AWS zones

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-4126

This PR adds the docs about creating a hosted cluster on AWS in multiple zones from [upstream ](https://hypershift-docs.netlify.app/how-to/aws/create-aws-hosted-cluster-multiple-zones/) to the official docs.

### CODE REVIEW

**Comment by lahinson** [comment]
@rokej When you can, PTAL at this PR. It ports over the upstream docs about creating a hosted cluster in multiple zones. Thank you!

File: clusters/hosted_control_planes/managing_hosted_aws.adoc [hunk 1]
```diff
@@ -46,6 +47,39 @@ When you review your information and optionally customize it before creating the
...
+<1> The `--zones` flag must specify availability zones within the region that is specified by the `--region` flag. The `--zones` flag is also available on the `hypershift create infra aws` command that is used to create infrastructure separately.
+
+The following per-zone infrastructure is created for all specified zones:
+
+* Public subnet
+* Private subnet
+* NAT gateway
+* Private route table (public route table is shared across public subnets)
+
+Two `NodePool` resources are created, one for each zone. The node pool name is suffixed by the zone name. The private subnet for zone is set in `spec.platform.aws.subnet.id`.
```
Comment (rokej) [inline clusters/hosted_control_planes/managing_hosted_aws.adoc]: Two `NodePool` resources are created, one for each zone.  

Should this be changed to the following indicating that one nodepool is created for each zone specified?

A different `NodePool` resource is created for each zone.

Comment (lahinson) [reply clusters/hosted_control_planes/managing_hosted_aws.adoc]: Yes, that's clearer. Good suggestion!

Comment (lahinson) [reply clusters/hosted_control_planes/managing_hosted_aws.adoc]: @rokej I updated the text to "One `NodePool` resource is created for each zone."

Thanks!

**Review by rokej — APPROVED** [review]

**Review by oafischer — APPROVED** [review]
Looks great, thanks!

**Pull request merged**